VAMobile/src/components/Templates/FeatureLandingAndChildTemplate.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { context, render, screen, fireEvent } from 'testUtils'
+import { FeatureLandingTemplate } from './FeatureLandingAndChildTemplate'
+import TextView from 'components/TextView'
+
+context('FeatureLandingAndChildTemplate', () => {
+    const onPressSpy = jest.fn()
+    const onBackPressSpy = jest.fn()
+
+    const initializeTestInstance = (
+        headerButton?: any,
+        footerContent?: React.ReactNode
+    ) => {
+        render(<FeatureLandingTemplate backLabel='Back' backLabelOnPress={onBackPressSpy} title='Title' headerButton={headerButton} footerContent={footerContent} />)
+    }
+
+    beforeEach(() => {
+        initializeTestInstance()
+    })
+
+    it('should render the title as a header when passed in', () => {
+        initializeTestInstance()
```

Comment:
D'oh!! Sorry, forgot I changed the test because the title is required, I was initializing it either with or without a title and forgot to go back and remove those. Updated.